### PR TITLE
Fix TRD digit constructor w/o channel ID

### PR DESCRIPTION
**Pull request by martenole**
@bazinski would this be OK?

@tdietel could you add me to the list of code owners for the TRD such that I get automatic notifications about code changes?

Cheers,
Ole

### CODE REVIEW

**Review by bazinski — APPROVED** [review]
I have those changes as well, in anycase will add the tests once finished in a seperate one.

File: Detectors/TRD/base/include/TRDBase/Digit.h [hunk 1]
```diff
@@ -72,10 +72,10 @@ class Digit
...
 
  private:
-  std::uint16_t mDetector{0}; // detector, the chamber
-  std::uint8_t mROB{0};       // read out board with in chamber
-  std::uint8_t mMCM{0};       // MCM chip this digit is attached to
-  std::uint8_t mChannel{0};   // channel of this chip the digit is attached to.
+  std::uint16_t mDetector{0}; // chamber ID [0..539]
+  std::uint8_t mROB{0};       // readout board ID within mDetector [0..7]
+  std::uint8_t mMCM{0};       // MCM chip ID on top of mROB [0..15]
+  std::uint8_t mChannel{0};   // TRAP ADC channel for mMCM [0..20], note: conversion to pad column described in https://cds.cern.ch/record/1973326 fig. 4.8 (TODO: put correct reference to ROB_MCM_numbering.pdf)
```
Comment (bazinski) [inline Detectors/TRD/base/include/TRDBase/Digit.h:78]: I have added the pic to the TDP. pr pending my gitlab struggles.

**Comment by martenole** [comment]
@bazinski if it is easier for you I can of course close this one again. Should I do that and will you open a PR?

**Review by bazinski — APPROVED** [review]

**Comment by bazinski** [comment]
@martenole you can leave this in, will simply add my unit tests in a seperate pr so this never goes awry again.

**Comment by bazinski** [comment]
@martenole  can you cherry pick from https://github.com/bazinski/AliceO2/commit/c0026594cb55b1a4ca8ba8a7526247aac0b62390
or would prefer i do a seperate pr ?
At least the testing showed something else wrong so not in vain.

**Comment by martenole** [comment]
@bazinski your changes look ok to me. I would then simply close this one. But there is no PR yet, right?

**Comment by martenole** [comment]
Or why should I cherry-pick? I don't see any changes in here that you don't have as well

**Comment by bazinski** [comment]
@martenole maybe cherry pick is the wrong term? there were changes to 2 of the constructors to add setting the detector, and then the tests.

**Comment by martenole** [comment]
Closing, as these changes will come in a separate PR from Sean

**Pull request closed without merging**